### ISSUE TITLE
판례에 링크를 걸때...

### ISSUE THREAD

**Issue by mkh2009** [no05s/pdf-support#22]
![20190620_190227](https://user-images.githubusercontent.com/51990901/59840905-498b0f00-938e-11e9-80a1-db9903d8b6ba.png)
 
판례에서 사건번호 뒤에  '등' 도 같이 링크거나요?

**Comment by dhkim11** [comment]
네. 판결유형 앞에 '등' ( ~등 판결)이 언급되었으므로 '대법원 ~ 등 판결' 까지 링크범위로 잡아 작업해주시면 됩니다.
